jvm/src/main/scala/uk/gov/homeoffice/drt/jsonformats/PassengersSummaryFormat.scala

```diff
@@ -0,0 +1,59 @@
+package uk.gov.homeoffice.drt.jsonformats
+
+import spray.json.{DefaultJsonProtocol, JsArray, JsNumber, JsObject, JsString, JsValue, RootJsonFormat, enrichAny}
+import uk.gov.homeoffice.drt.models.{PassengersSummaries, PassengersSummary}
+import uk.gov.homeoffice.drt.ports.Queues
+import uk.gov.homeoffice.drt.ports.Queues.Queue
+import uk.gov.homeoffice.drt.time.LocalDate
+
+import scala.collection.immutable.SortedMap
+
+object PassengersSummaryFormat extends DefaultJsonProtocol {
```

Comment:
Just other way , I would make LocalDateJsonFormat as trait and extend in PassengersSummaryFormat and make one of JsonFormat to JsonFormatDate or JsonFormatPassengerSummary .  Not sure if there is any advantage of one over other implementation .